rules/do-you-know-the-3-steps-to-a-pbi/rule.md

```diff
@@ -1,74 +1,67 @@
+
 ### 1. Ready
 
-1. Take the next PBI that was approved by the 
-      [Product Owner](/rules-to-better-product-owners)
-2. Is the PBI ready?
-Check your PBI against your 
-      [Definition of Ready](/have-a-definition-of-ready). "Ready" PBIs must have Acceptance Criteria. A good Definition of Ready also encourages a test-first mentality in requirements e.g. Use Spec Flow (Given, When, Then) and/or create Test Cases and Test Steps first.
-3. Attach and copy the Product Owner's email into the Acceptance Criteria (as per [Do you attach and copy emails to the PBI?](/do-you-attach-emails-to-the-pbi)).
-4. Email notification to the Product Owner - 
-      [use the @ mention rule](/when-you-use-mentions-in-a-pbi)
-5. Break down your PBI into tasks.
-6. Don't forget to make a task for testing! (So that it is visible in the task board). Note: You can also 
-      [customize the kanban board](https://www.visualstudio.com/en-us/get-started/work/work-from-the-kanban-board-vs) by adding a new column for testing, but we recommend adding a testing task to the PBI instead.
+1. Take the next PBI that was approved by the [Product Owner](/rules-to-better-product-owners)
+2. Email notification to the Product Owner -  [use the @ mention rule](/when-you-use-mentions-in-a-pbi)
+3. Break down your PBI into tasks
+
+::: info
+Don't forget to make a task for testing! (So that it is visible in the task board). Note: You can also [customize the kanban board](https://www.visualstudio.com/en-us/get-started/work/work-from-the-kanban-board-vs) by adding a new column for testing, but we recommend adding a testing task to the PBI instead.
+:::
 
-::: bad  
-![Figure: Adding a new "Test" state. This is only visible in the Product Backlog and not the Sprint Backlog](KB-customize-board-columns.png)  
+::: bad
+![Figure: Adding a new "Test" state. This is only visible in the Product Backlog and not the Sprint Backlog](KB-customize-board-columns.png)
 :::
 
-::: good  
-![Figure: Testing Task added to PBI. This is the board the team will use for 90% of the Sprint, so testing should be clearly visible here](Testing-task.png)  
+::: good
+![Figure: Testing Task added to PBI. This is the board the team will use for 90% of the Sprint, so testing should be clearly visible here](Testing-task.png)
 :::
 
 ### 2. Code
 
 1. From the PBI, create a new branch (so that your work is automatically tagged to the PBI)
 2. On the Kanban Board, move your Task into "In Progress"
-3. Checkout your new branch and start coding!
-4. Code, code, code… (Red, Green, Refactor)
-5. Push your changes, open a Pull Request
-6. If you want feedback early, record a Done Video. E.g. Snagit
-
+3. Checkout your new branch and code, code, code… (Red, Green, Refactor)
```

Comment:
What is red, green, refactor....no idea

Reply:
its a TDD practice, the rule doesn't explain it entirely correctly but its worth linking to either way

Reply:
Can we link a rule or something? Just so people can find out what it means.

Reply:
Oh you've already done that, silly me!